src/ViewModels/test/TasExplorerViewModelTests.cs

```diff
@@ -980,32 +980,19 @@ public void SettingAuthenticationRequiredToFalse_DoesNotExpandCfInstanceViewMode
         }
 
         [TestMethod]
-        [TestCategory("DeleteConnection")]
-        public void DeleteConnection_SetsTasConnectionToNull_WhenArgIsCfInstanceViewModel()
+        [TestCategory("LogOutTas")]
+        public void LogOutTas_SetsTasConnectionToNull_WhenArgIsCfInstanceViewModel()
         {
             _sut.TasConnection = _fakeTasConnection;
 
             Assert.IsNotNull(_sut.TasConnection);
             Assert.IsTrue(_sut.TasConnection is CfInstanceViewModel);
 
```

Comment:
Now that the `LogOutTas` method no longer checks the type of the argument to make sure it's a `CfInstanceViewModel` I don't think these tests should mention or do anything with `CfInstanceViewModel`s.

```suggestion
        public void LogOutTas_SetsTasConnectionToNull_AndSetsIsLoggedInToFalse()
        {
            _sut.TasConnection = _fakeTasConnection;

            Assert.IsNotNull(_sut.TasConnection);
            Assert.IsTrue(_sut.IsLoggedIn); // NOTE: idk if this will currently pass or not; might have to change test setup
```